cmd/crictl/container.go

```diff
@@ -351,7 +351,12 @@ var removeContainerCommand = &cli.Command{
 		}
 
```

Comment:
should images show help in here: https://github.com/kubernetes-sigs/cri-tools/blob/cd4612fba769342cdde32cdc32ce817103645b90/cmd/crictl/image.go#L396 for consistency?

Reply:
Done